src/data-requests/slack.js

```diff
@@ -0,0 +1,99 @@
+// Slack data request
+import fetch from 'isomorphic-unfetch'
+import { links } from 'config'
+import getIP from 'helpers/getIP'
+
+// posts to slack (configured in CCDL channel) and returns true if 200, otherwise false
+const postToSlack = async (params) => {
+  try {
+    const res = await fetch(
+      // TEMP: usng the webhook url for the ccdl test channel for dev
+      process.env.SLACK_HOOK_URL || process.env.NEXT_PUBLIC_SLACK_HOOK_URL,
```

Comment:
We cannot have a slack hook be available as `NEXT_PUBLIC`. Message me on slack if you need more details. But for now just evaluate that string in `api/data-requests.js` and then pass it to `submitSlackDataRequest`

Reply:
For the evaluation of `process.env.SLACK_HOOK_URL`, I've updated `submitSlackDataRequest` to accept a new parameter `hookUrl`, and passing this env from `api/data-requests.js` when calling this function. Pls let me know your feedback, thank you David!  

See commit at 6d3670af39f6639aedc39d85602333e27b9a3116.